BB-SC-TC/Testing/BB_Shadowing.md

```diff
@@ -0,0 +1,75 @@
+
+# Shadowing
+
+## BB Tags(s)
+<!-- Tag(s) define in which area(s) (cloud, in-vehicle) the BB is executed, and what type of BB it is (tool, process, microservice) -->
+BB-SC-TC
+
+## Functional Clusters
+<!-- In which Functional Cluster the BB be located; if none of the existing fit new required -->
+Testing
+
+## Layer
+<!-- AppLayer, MWLayer, OSLayer, HWLayer -->
+MWLayer
+
+## Known Implementation
+
+## ID (unique name)
+
+## Description
+<!-- General Description of the BB -->
+A system working passively in the background that uses recorded data and input from sensors and cameras in the car to make its own hypothetical driving decisions. These hypothetical driving decisions are then compared to the decisions the (human) driver of the car makes. Both the recorded data and the comparison are used, amongst others, to discover unthought of edge and corner cases, and to evaluate and demonstrate the safety of autonomous functionalities.
+Proposal for standardization of recording services, comparison services, and process execution services are standardized in conjunction data collector and orchestrator. The rest seems to be added value of the functionality.
+
+## Rationale
+<!-- Explanation why we need the BB; what problem want to be solved -->
+The use of shadowing during development (and potentially even during operation) of vehicles can enhance the ADAS algorithm/configuration development which will otherwise mostly be based on artificial test data and test drives. Shadowing will support the improvement of the vehicles in the field and enhance the safety of the users.
+
+## Governance Applicable S-BB(s)
+<!-- Reference to e.g. UN/EU CRA Cyber Resilience Act; UNECE 156 - Software update and software update management system
+Reference to defined S-BB(s) 
+Reference to e.g. IS026262, AUTOSAR Spec. X -->
+TDB
+
+## Compose BB(s)
+<!-- Link to required BB(s) 
+E.g. BB-SC StateManagement 
+BB is a composition of other BBs -->
```

Comment:
Isn't this a composition of e.g. Digital Twin + Persistency?

Reply:
No it is not. Literature suggests digital Shadowing as a real-time active object. Needs further discussion.

A digital modelis a virtual representation of a physical object, system, or process.
A digital shadow mirrors the current state and behavior of a physical entity or system.

Reply:
suggestion: extend description accordingly